doc/api/stream.md

```diff
@@ -2077,6 +2077,27 @@ Calling `Readable.from(string)` or `Readable.from(buffer)` will not have
 the strings or buffers be iterated to match the other streams semantics
 for performance reasons.
 
+### `stream.Readable.concat(streams, [options])`
+
+* `streams` {Readable[]|Iterable[]|AsyncIterable[]} List of readable streams or iterator of readable stream generator
+* `options` {Object} Options provided to `new stream.Readable([options])`.
+  By default, `Readable.concat()` will set `options.objectMode` to `false`, unless
+  this is explicitly opted out by setting `options.objectMode` to `true`.
+* Returns: {stream.Readable}
+
+A utility method to concatenate multiple readable streams to single one. The main readable stream will consume the
+list of readable streams in order
```

Comment:
```suggestion
A utility method to concatenate multiple readable streams to single one. The
main readable stream will consume the list of readable streams in order.
```
Every line must be 80 characters or less in length and this line must end with a period.